adapters/handlers/rest/clusterapi/indices.go

```diff
@@ -364,7 +365,14 @@ func (i *indices) postObjectBatch(w http.ResponseWriter, r *http.Request,
 		return
 	}
 
-	errs := index.IncomingBatchPutObjects(r.Context(), shard, objs)
+	var schemaVersion uint64
```

Comment:
Could you refactor the code for reading the version into a function to make it reusable